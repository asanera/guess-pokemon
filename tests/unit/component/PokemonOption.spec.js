import PokemonOption from "@/components/PokemonOption";
import { shallowMount } from "@vue/test-utils";
import { pokemons } from "../mocks/pokemons.mock";

describe('Test PokemonOption', () => {

    let wrapper
    beforeAll(() => {
         wrapper = shallowMount(PokemonOption, {
            props: {
                pokemons
            }
         });
    });

    test('Debe hacer match con el snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    })
})