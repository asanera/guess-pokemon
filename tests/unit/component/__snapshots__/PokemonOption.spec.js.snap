// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PokemonOption Debe hacer match con el snapshot 1`] = `
<div
  class="options-container"
>
  <ul>
    
    <li>
      pikachu
    </li>
    <li>
      ivysaur
    </li>
    <li>
      venusaur
    </li>
    <li>
      charmander
    </li>
    
  </ul>
</div>
`;
